sys-cluster/hpx/files/hpx-1.5.1-cmake.patch

```diff
@@ -0,0 +1,32 @@
+From 8bee06baabf24c4b8f48766dba5c76d1979c8ea6 Mon Sep 17 00:00:00 2001
+From: Kurt Kanzenbach <kurt@kmk-computers.de>
+Date: Sat, 19 Sep 2020 13:19:42 +0200
+Subject: [PATCH] tests: Remove unused cmake options
+
+This fixes the following warnings:
+
+| * One or more CMake variables were not used by the project:
+| *   BOOST_ROOT
+| *   SETUP_TYPE
```

Comment:
This remains unfixed, but I'm not going to hold it back due to it.

Reply:
Yeah, true. Some tests use `SETUP_TYPE` and some don't. IMO it's a false positiv.